synapse/federation/federation_client.py

```diff
@@ -238,13 +238,15 @@ async def claim_client_keys(
         self,
         destination: str,
         query: Dict[str, Dict[str, Dict[str, int]]],
+        user: UserID,
         timeout: Optional[int],
     ) -> JsonDict:
         """Claims one-time keys for a device hosted on a remote server.
 
         Args:
             destination: Domain name of the remote homeserver
             content: The query content.
+            user: The user id of the requesting user
```

Comment:
I think we usually put the `user` as one of the early arguments (my brain really wants it to be `user`, `destination`, `query`).

It doesn't really matter though and we can leave it, but I'm curious if there's a reason you chose to put it in the location it is in?

Reply:
I'm happy to change it if that's the general pattern - I didn't have much reason to put it where it is other than I generally default to adding new parameters to the end of a parameter list.

Reply:
It is a required parameter anyway, so I don't think adding it at the end matters much. (And this isn't even at the end!)

Reply:
Sorry I meant end of the required parameters - but thank you for pointing this out, this is the sort of convention I am trying to internalize so I appreciate knowing about it :)